nav2_costmap_2d/plugins/obstacle_layer.cpp

```diff
@@ -158,6 +158,8 @@ void ObstacleLayer::onInitialize()
     declareParameter(source + "." + "obstacle_min_range", rclcpp::ParameterValue(0.0));
     declareParameter(source + "." + "raytrace_max_range", rclcpp::ParameterValue(3.0));
     declareParameter(source + "." + "raytrace_min_range", rclcpp::ParameterValue(0.0));
+    declareParameter(source + "." + "point_cloud_transport",
```

Comment:
Add to configuration files the population of this parameter with a commented out bit on the available options for consideration